sympy/physics/mechanics/tests/test_eombase.py

```diff
@@ -0,0 +1,30 @@
+import sympy.physics.mechanics.eombase as eombase
+
+# Determine what inputs should be used
+EOM = eombase.EOMBase(frame=None)
+
+# Should contain access to all the property attributes contained in existing EOM
+# generators
+
+Thing = "Place holder until a correct output is chosen"
+
+assert EOM.auxiliary_eqs == Thing
+assert EOM.bodylist == Thing
```

Comment:
I think `bodies` is a better name. It probably should default to a tuple, not a list and it should work with any iterable.